server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tservers/TabletServerResource.java

```diff
@@ -141,28 +146,27 @@ public TabletServersRecovery getTserverRecovery() {
   /**
    * Generates details for the selected tserver
    *
-   * @param tserverAddr
+   * @param tserverAddress
    *          TServer name
    * @return TServer details
    */
   @Path("{address}")
   @GET
-  public TabletServerSummary getTserverDetails(@PathParam("address") String tserverAddr) throws Exception {
-
-    String tserverAddress = tserverAddr;
+  public TabletServerSummary getTserverDetails(@PathParam("address") String tserverAddress) throws Exception {
```

Comment:
+1 I prefer full english words for parameter variable names (especially with Strings).  Unless you are using VIM as your IDE, abbreviation doesn't help anyone and only leads to confusion.  Granted, this example 99% of developers will know what "addr" means but who knows... there could be another variable to come along with the name tserverAdder or something.